src/test/java/org/humancellatlas/ingest/IngestStateTrackingApplicationTests.java

```diff
@@ -251,6 +251,123 @@ public void testSuccessfulEventRunthroughWithoutArchiving() {
         assertEquals(SubmissionState.COMPLETE, state);
     }
 
+    @Test
+    public void testSuccessfulEventRunthroughAfterUpdate() {
```

Comment:
IS this test testing one thing?

Reply:
Yes, it's testing the flow from DRAFT -> EXPORTED, back to DRAFT and then to EXPORTED again. I don't think you can break this down any further. It's the whole reason this bug happened